backend/api/documents.py

```diff
@@ -0,0 +1,132 @@
+from typing import Any, Dict, Tuple, cast
+
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from backend.database_handler_entity import mongo
+from backend.document_status import Status
+from backend.role import Role
+
+document_api = Blueprint('document_api', __name__)
+
+
+@document_api.route('/document', methods=["POST"])
+@jwt_required()
+def create_document() -> Tuple[Any, int]:
+    document_name = request.data.decode("utf-8")
+    document_identifier = mongo.create_document(document_name, get_jwt_identity())
+
+    if document_identifier is None:
+        return jsonify({"message": "Document already exist!"}), 403
+
+    return jsonify(str(document_identifier)), 201
+
+
+@document_api.route('/document/<identifier>', methods=["PUT", "GET", "DELETE"])
+@jwt_required()
+def update_document_content(identifier: str) -> Tuple[Any, int]:
+    if request.method == "GET":
```

Comment:
I think for this kind of `if - elif - else` constructions would be better to use separate functions for every condition to increase readability and decrease lines of code in function. Example:
```
if request.method == "GET":
    get_document(...)
elif request.method == "PUT":
    update_document(...)
else:
    ...
```